test_conformance/spirv_new/test_op_spec_constant.cpp

```diff
@@ -0,0 +1,162 @@
+/******************************************************************
+Copyright (c) 2020 The Khronos Group Inc. All Rights Reserved.
+
+This code is protected by copyright laws and contains material proprietary to
+the Khronos Group, Inc. This is UNPUBLISHED PROPRIETARY SOURCE CODE that may not
+be disclosed in whole or in part to third parties, and may not be reproduced,
+republished, distributed, transmitted, displayed, broadcast or otherwise
+exploited in any manner without the express prior written permission of Khronos
+Group. The receipt or possession of this code does not convey any rights to
+reproduce, disclose, or distribute its contents, or to manufacture, use, or sell
+anything that it may describe, in whole or in part other than under the terms of
+the Khronos Adopters Agreement or Khronos Conformance Test Source License
+Agreement as executed between Khronos and the recipient.
+******************************************************************/
+
+#include "testBase.h"
+#include "types.hpp"
+
+
+template <typename T>
+int run_case(cl_device_id deviceID, cl_context context, cl_command_queue queue,
+             const char *name, T init_buffer, T spec_constant_value,
+             T final_value, bool use_spec_constant,
+             bool (*notEqual)(const T &, const T &) = isNotEqual<T>)
+{
+    clProgramWrapper prog;
+    cl_int err = CL_SUCCESS;
+    err = get_program_with_il(prog, deviceID, context, name);
```

Comment:
As I previously explained, this does not work, as `get_program_with_ir()` calls `clBuildProgram()`.